src/locales/en.json

```diff
@@ -117,31 +117,31 @@
     },
     "risk_tolerance_1_percent_label": "Standard Caution Budget",
-    "risk_tolerance_1_percent_explanation": "Budget: 1% chance of COVID per year (200 microCOVIDs per week) (suggested for healthy people NOT in close contact with more vulnerable people)",
+    "risk_tolerance_1_percent_explanation": "Budget: 1% chance of COVID per year (200 mCs per week) (suggested for healthy people NOT in close contact with more vulnerable people)",
     "risk_tolerance_point1_percent_label": "High Caution Budget",
-    "risk_tolerance_point1_percent_explanation": "Budget: 0.1% chance of COVID per year (20 microCOVIDs per week) (suggested if you or your close contacts are more vulnerable to COVID)",
+    "risk_tolerance_point1_percent_explanation": "Budget: 0.1% chance of COVID per year (20 mCs per week) (suggested if you or your close contacts are more vulnerable to COVID-19)",
     "intro": {
-      "whats_this": "<p>We’ve constructed a calculator that lets you estimate the risk of getting COVID from a wide range of activities, using the <1>best research available</1>. We hope this tool will help hone your intuition, lower your stress levels, and figure out good harm-reduction strategies.</p><p>The first step in protecting your family and community is to avoid getting COVID yourself.</p>",
+            "whats_this": "<p>The calculator sums up the risk of getting COVID-19 from a wide range of activities, using the <1>best research available</1>. This in the hopes of helping you hone your intuition, figuring out good harm-reduction strategies, and lowering your stress levels.</p><p>The first step in protecting your family and community is to avoid getting COVID yourself.</p>",
       "call_to_action": "Calculate the approximate COVID risk of any activity"
     },
     "risk_step_label": "Step 2: Describe the activity",
     "type_of_interaction": "Is this a single activity or an ongoing relationship?",
     "risk_group_empty_warning": "First, enter your location",
     "warning": {
       "important": "Important",
-      "body": "In this tool we state our best estimate based on available evidence, even when that evidence is not conclusive. We have read a lot of experts' research, but we are not ourselves experts in this topic. This work has not been scientifically peer-reviewed. There is still a lot of uncertainty about COVID. Do not rely on this tool for medical advice. Please continue to follow government guidance."
+      "body": "The estimate is based on available (and even unconclusive) evidence the microCOVID authors are not experts in. This work is not scientifically peer-reviewed, and you should not rely on it for medical advice. Please continue following the guidance of your government, which the microCOVID authors, or translators aren't responsible for either."
```

Comment:
This is getting pretty passive aggressive. Could you tone it down?

Reply:
The most important part is mentioning translators are not liable here, there is a combo at the end to that effect. The last sentence is just what is already there. "Pay attention to the guidance of your government." is probably better.

The estimate is based on available (and even inconclusive) evidence the microCOVID authors are not experts in. This work is not scientifically peer-reviewed, and you should not rely on it or the correctness of translations thereof for medical advice. Pay attention to the guidance of your government.